docs/v0.3.0/design/ratelimit.md

```diff
@@ -172,7 +172,7 @@ spec:
 
 Here's an example highlighting this -
 
-```
+```yaml
```

Comment:
cc @Xunzhuo , will this work?

Reply:
Good to go, tested locally.

<img width="1078" alt="image" src="https://user-images.githubusercontent.com/48784001/233064313-cf5c30ba-1732-4728-942f-5637b5bcdadc.png">